types/errors.go

```diff
@@ -123,6 +124,7 @@ func HandleServerError(writer http.ResponseWriter, err error) {
 	case *ForbiddenError:
 		respErr = responses.SendForbidden(writer, err.Error())
 	default:
+		log.Error().Err(err).Msg(handleServerErrorStr)
```

Comment:
Do we want to generate two identical log lines in WARN and ERROR level?

Reply:
I've just pushed some changes in order not to duplicate the log lines, PTAL